### ISSUE TITLE
fn trivial-casts: String.to_string()

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/rust#54091]
````rust
#![warn(trivial_casts)]
fn main() {
	let x: u32 = 4;
	let _s2: String = format!("{}", x).to_string().to_string(); // should warn
	let _s3 = String::from("y").to_string(); // should warn
}
````
Currently there is no warning emitted at all.
`rustc 1.30.0-nightly (0198a1ea4 2018-09-08)`

**Comment by matthiaskrgr** [comment]
I'll try to make this a clippy lint...
